build/docker/Dockerfile.compiler

```
@@ -4,8 +4,12 @@ ARG OS=ubuntu
 FROM falkordb/falkordb-build:$OS AS builder
 
+ARG DEBUG=0
+
 WORKDIR /FalkorDB
 
 COPY . /FalkorDB
 
-RUN make
+RUN make DEBUG=$DEBUG
+
+COPY --from=builder /FalkorDB/bin/linux*/src/ /FalkorDB/bin/
```

Hadolint (2.12.0)

[error] `COPY --from` cannot reference its own `FROM` alias

(DL3023)